From b8f4831f45252191aa303d9e0d4171db739f68d0 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Wed, 29 May 2019 10:34:09 +0000
Subject: [PATCH] arm64: dts: dontbeevil: remove pull-up bias on touch_intr_pin

Currently some touch "release" events are not sent by the kernel.
Removing this biais helps getting all the touchscreen events correctly.

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-dontbeevil.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-dontbeevil.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-dontbeevil.dts
index 24c32821572f..98d7fd13b583 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-dontbeevil.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-dontbeevil.dts
@@ -259,8 +259,6 @@
 &pio {
        touch_intr_pin: touch_intr_pin {
                pins = "PH4";
-               // TODO: perhaps remove this bias
-               bias-pull-up;
                function = "irq";
        };
 };
-- 
2.21.0

